src/constants/styling/buttons.scss

```diff
@@ -33,6 +35,8 @@
   font-weight: 600;
   padding: 5px 15px;
   border-radius: 5px;
+  margin-bottom: 3px;
+  margin-left: 0.5rem;
```

Comment:
These changes you have made will affect all secondary buttons. I don't think there are as many of these. Have you checked that the alignment of other buttons is still okay?